pkg/api/kubegraph/analysis/pod_test.go

```diff
@@ -38,6 +39,20 @@ func TestRestartingPodWarning(t *testing.T) {
 		t.Fatalf("expected %v, got %v", e, a)
 	}
 
+	sort.Sort(osgraph.ByNodeID(markers))
+	if strings.Index(markers[0].Message, "container ") != 0 {
```

Comment:
I was trying to do something like `strings.StartsWith` in case the word `container` occurs elsewhere.